action/src/main/java/com/adyen/checkout/action/ActionHandlingComponent.kt

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2022 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by oscars on 11/11/2022.
+ */
+
+package com.adyen.checkout.action
+
+import android.app.Activity
+import android.content.Intent
+import androidx.annotation.RestrictTo
+import com.adyen.checkout.components.model.payments.response.Action
+import com.adyen.threeds2.customization.UiCustomization
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+interface ActionHandlingComponent {
```

Comment:
Can we add the docs for the other methods as well? We can just copy them from other classes I think. Or i wonder if we should reuse this interface with the other action components to avoid duplication 🤔

Reply:
Not sure how we would do the `observe` and `removeObserver` methods from `ActionComponent`. I think it's better to keep them separate as it feels like they have different purposes

Reply:
Added docs for now

Reply:
👍